internal/infrastructure/mongo/auth_request_test.go

```diff
@@ -0,0 +1,72 @@
+package mongo
+
+import (
+	"context"
+	"testing"
+
+	"github.com/caos/oidc/pkg/oidc"
+	"github.com/reearth/reearth-backend/pkg/auth"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_authRequestRepo_FindByID(t *testing.T) {
```

Comment:
```suggestion
func TestAuthRequestRepo(t *testing.T) {
```

Collect name is `TestXXX_YYY`, not `Test_XXX_YYY`, but this test includes unit tests for `Save` method also. So I suggest that name.

Reply:
@rot1024 
Thank you, I fixed it.